### DISCUSSION TITLE
AWS Device Farm

### DISCUSSION THREAD

**Discussion by yerx** [wix/Detox#2875] [General]
Are there any plans to make Detox compatible with AWS's Device Farm?

**Comment by d4vidi** [comment]
None ATM, and we're not currently aware of how AWS' platform is built. Having said that - one thing that might help plan such a thing is if you could quickly review [Detox' architecture](https://github.com/wix/Detox/blob/master/docs/Introduction.HowDetoxWorks.md#architecture) and try to spell out the list of gaps for us to get there 🙏🏻
